syntax = "proto3";

package rules;

message BreakoutRequest {
    string symbol = 1;
    int32 lookback = 2;
    bool use_attenuation = 3;
    float scaling_factor = 4;
    string scaling_type = 5;

}

message BreakoutResponse {
    bytes series = 1;
}

service Breakout{
    rpc get_breakout (BreakoutRequest) returns (BreakoutResponse);
}